src/core/RecyclerListView.tsx

```diff
@@ -579,11 +579,11 @@ export default class RecyclerListView<P extends RecyclerListViewProps, S extends
 
     private _onScroll = (offsetX: number, offsetY: number, rawEvent: ScrollEvent): void => {
         //Adjusting offsets using distanceFromWindow
-        let scrollCorrection = 0;
-        if (this.props.correctedScrollOffset) {
-            scrollCorrection = this.props.correctedScrollOffset();
+        let correctionDelta = 0;
+        if (this.props.scrollOffsetCorrectionDelta) {
+            correctionDelta = this.props.scrollOffsetCorrectionDelta();
         }
-        this._virtualRenderer.updateOffset(offsetX, offsetY, -scrollCorrection, true);
+        this._virtualRenderer.updateOffset(offsetX, offsetY, correctionDelta, true);
```

Comment:
`-` missing?

Reply:
If not, can you update with a comment why this call was taken?

Reply:
Added

Reply:
From contract we should take the entire new offset externally